man/systemd.exec.xml

```diff
@@ -366,6 +366,22 @@
         <xi:include href="version-info.xml" xpointer="v233"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>BindJournalSockets=</varname></term>
+
+        <listitem><para>Takes a boolean argument. If true, sockets from <citerefentry>
+        <refentrytitle>systemd-journald.socket</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+        will be bind mounted into the mount namespace. This is particularly useful when a different rootfs
```

Comment:
`bind mounted` -> `bind-mounted` ?
(You know, my English is bad, feel free to ignore.)

Reply:
I think either way should work?

Reply:
OK.